library/src/data/cities/build.ts

```diff
@@ -0,0 +1,22 @@
+import { CityProto, CityArrayProto } from "./City.pb";
+import cities from "all-the-cities";
+import fs from "fs";
+
+// We reduce a bit the number of cities to speed-up the lookup,
+// which is using the levenshtein distance.
+
+export const buildCitiesAbove10k = () => {
+  const filteredCities: CityProto[] = cities
+    .filter((city) => city.population >= 10_000)
+    .map((city) => ({
+      ...city,
+      name: city.name,
+      location: { lat: city.loc.coordinates[1], lng: city.loc.coordinates[0] },
+      country: city.country,
+    }));
+
+  fs.writeFileSync(
+    `${__dirname}/cities-above-10k.pbf`,
```

Comment:
I usually try to avoid building paths manually as it can change depending on the platform (imagine some people try to use this on Windows 😬)

```suggestion
    join(__dirname, 'cities-above-10k.pbf'),
```

with `import { join } from 'path';`

Reply:
This comment is actually relevant for other places where you manipulate paths

Reply:
Ok, but I will not refactor `copy-package-json.mjs` until we decide (and test) that we are compatible with Windows.

Reply:
Fair enough haha